server/controllers/groups.js

```diff
@@ -419,37 +420,45 @@ module.exports = function(app) {
    */
   const createFromGithub = (req, res, next) => {
 
-      var payload = req.required.payload;
-      const connectedAccountId = req.jwtPayload.connectedAccountId;
+    var payload = req.required.payload;
+    const connectedAccountId = req.jwtPayload.connectedAccountId;
 
-      var creator, options;
-      const group = payload.group;
-      const contributors = payload.users;
-      const creatorGithubUsername = payload.github_username;
-      var dbGroup;
+    var creator, options;
+    const group = payload.group;
+    const githubUser = payload.user;
+    const contributors = payload.users;
+    const creatorGithubUsername = payload.github_username;
+    var dbGroup;
 
-      ConnectedAccount
-        .findOne({
-          where: { id: connectedAccountId },
-          include: { model: User }
-        })
-        .tap(ca => {
-          creator = ca.User;
-          options = {
-            role: roles.MEMBER,
-            remoteUser: creator
-          };
-        })
-        .then(() => Group.findOne({where: {slug: group.slug.toLowerCase()}}))
-        .then(existingGroup => {
-          if (existingGroup) {
-            group.slug = `${group.slug}+${Math.floor((Math.random() * 1000) + 1)}`;
-          }
-          return Group.create(group)
-        })
-        .tap(g => dbGroup = g)
-        .then(() => Activity.create({
-          type: activities.GROUP_CREATED,
+    ConnectedAccount
+      .findOne({
+        where: { id: connectedAccountId },
+        include: { model: User }
+      })
+      .tap(ca => {
+        creator = ca.User;
+        options = {
+          role: roles.MEMBER,
+          remoteUser: creator
+        };
+      })
+      .tap(() => {
+        if (githubUser) {
+          creator.website = githubUser.blog;
+          creator.name = githubUser.name;
+          return creator.save();
+        }
+      })
+      .then(() => Group.findOne({where: {slug: group.slug.toLowerCase()}}))
```

Comment:
`toLowerCase()` is normally redundant, `Group.js` already has a validator to make slug lower-case.